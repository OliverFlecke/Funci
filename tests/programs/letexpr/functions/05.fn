-- test environment

main :: Int =
    let f x x x x x :: Int -> Int -> Int -> Int -> Int -> Int = (2 * x) 
     in f 0 0 0 0 7
