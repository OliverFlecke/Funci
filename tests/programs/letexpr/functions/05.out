Number 14 (Unit [])